spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -835,7 +843,11 @@ public interface AnnotatedTypeConfigurator<T> {
 * `constructors()` returns a set of `AnnotatedConstructorConfigurator` to configure annotations on constructors.
 * `filterConstructors()` returns a `Stream<AnnotatedConstructorConfigurator>` filtered by applying the provided Predicate on `Constructors()`
 
-`AnnotatedMethodConfigurator` and `AnnotatedConstructorConfigurator` are accessed through `AnnotatedTypeConfigurator`:
+[[annotated_method_configurator]]
+
+==== `AnnotatedMethodConfigurator`
```

Comment:
Why is the first section named `AnnotatedTypeConfigurator SPI`, while the sections below (method, constructor,...)  lack the **SPI** keyword.
It really seems weird since they are all on the same level (all are equivalent subsection of `Alternative metadata sources`), I suppose you wanted to make it obvious that `AnnotatedTypeConfigurator` is the entry point for the whole thing? But that is already said in the text. I suggest we make the Annotated*Configurator chapters a subsection of `AnnotatedTypeConfigurator SPI`. That would IMO underline what is the entry point and also shed some light on how does the hierarchy work.

E.g. using current section numbers it would look like this:
11.4.1 `AnnotatedTypeConfigurator SPI`
 11.4.1.1 `AnnotatedMethodConfigurator`
 11.4.1.2 `AnnotatedConstructorConfigurator`
 11.4.1.3 `AnnotatedParameterConfigurator`
 11.4.1.4 `AnnotatedFieldConfigurator`
11.5  `Container lifecycle events`
...

WDYT?